core/config/toml/types.go

```diff
@@ -216,6 +260,91 @@ func (d *DatabaseSecrets) validateMerge(f *DatabaseSecrets) (err error) {
 	return err
 }
 
+type EthKey struct {
```

Comment:
Would it make sense to use `EVM` everywhere?
```suggestion
type EVMKey struct {
```

Reply:
agree that the skew is odd, but did it this way b/c the TOML for relayer is `EVM` while the keystore is Eth

so, it aligns with the existing convetions